HACKING.rst.txt

```diff
@@ -277,6 +277,12 @@ Therefore if you copied existing code with such a license or made a direct
 translation to Python of it, your code can't be included.  See also `license
 compatibility`_.
 
+A special case is the Numerical Recipes book: it's a book with arguably
+excellent explanations of numerical algorithms, useful as a reference, and that
+also has software that we can't use. Therefore, it is fine to refer to its
+*text* in the documentation, but we cannot accept *code* derived from Numerical
+Recipes.
```

Comment:
We more often cite it as a reference for users to read more about the algorithm than actually refer to it in the course of an implementation.